ai_coach/static/js/src/ai_coach.js

```diff
@@ -1,59 +1,86 @@
 /* Javascript for AICoachXBlock. */
-function AICoachXBlock(runtime, element) {
+function AICoachXBlock(runtime, element, args) {
+
+    let feedback_count = args.feedback_count;
+    const feedback_threshold = args.feedback_threshold;
 
     const studentAnswer = $('#student-answer', element);
-    const coachAnswer = $('.coach-answer p', element);
-    const messageElement = $('.messages', element);
+    const coachAnswerText = $('.coach-answer-text', element);
+    const coachAnswerContainer = $('.coach-answer-container', element);
+    const messageContainerElement = $('.message', element);
+    const messageTextElement = $('.message .message__title', element);
+    const tooltipElement = $('tool-tip', element);
+
+    function showSuccessMessage() {
+        messageContainerElement.addClass('bg-success');
+        messageContainerElement.removeClass('message--hide');
+        messageTextElement.text('Answer submitted successfully');
+    }
+
+    function showErrorMessage(error) {
+        messageContainerElement.addClass('bg-danger');
+        messageContainerElement.removeClass('message--hide');
+        messageTextElement.text(error);
+    }
+
+    function resetMessages() {
+        messageContainerElement.removeClass('bg-danger bg-success');
+        messageContainerElement.addClass('message--hide');
+        messageTextElement.text('');
+    }
+
+    function updateTooltipText() {
+        let tooltipText = 'You have reached the limit for asking help from coach'
```

Comment:
can we change this text to `You've exhausted all available chances to ask the coach for help.`